pipeline/filters/nest.md

```diff
@@ -109,6 +111,13 @@ $ bin/fluent-bit -i mem -p 'tag=mem.local' -F nest -p 'Operation=nest' -p 'Wildc
     Nest_under Memstats
     Remove_prefix Mem.
 ```
+{% endtab %}
+
+{% tab title="fluent-bit.yaml" %}
+```yaml
```

Comment:
The sample config is missing

Reply:
ping @nokute78

Reply:
Thank you for pointing it.
I added a missing example.